implement-shell-tools/wc/wc.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import os 
+import re
+
+parser = argparse.ArgumentParser(
+                    prog='wc command',
+                    description='word, line, character, and byte count')
+
+parser.add_argument("-c", "--bytes", help="The number of bytes in each input file is written to the standard output.", default = None, action="store_true")
+parser.add_argument("-l", "--lines", help="The number of lines in each input file is written to the standard output.", default = None, action="store_true")
+parser.add_argument("-w", "--words", help="The number of words in each input file is written to the standard output.", default = None, action="store_true")
+parser.add_argument("paths", nargs="+", help="The file to search")
+
+args = parser.parse_args()
+cwd = os.getcwd()
+
+
+def count_lines(content):
+    return len(content.splitlines())
+    
+def count_words(content):
+    return len(re.findall("[a-zA-Z\-\.'/]+", content))
```

Comment:
Whenever I run this solution, I see the following warning:
```
implement-shell-tools/wc/wc.py:22: SyntaxWarning: invalid escape sequence '\-'
  return len(re.findall("[a-zA-Z\-\.'/]+", content))
```

While the code seems to work even with the warning, could you elaborate on what could be the reason for the warning?